tests/pytorch/test_datasets_pt.py

```diff
@@ -434,3 +434,24 @@ def test_ic03(input_size, num_samples, rotate, mock_ic03_dataset):
     assert len(ds) == num_samples
     assert repr(ds) == f"IC03(train={True})"
     _validate_dataset(ds, input_size, is_polygons=rotate)
+
+
+def test_mjsynth_dataset(mock_mjsynth_dataset):
+    input_size = (32, 128)
+    ds = datasets.MJSynth(
+        *mock_mjsynth_dataset,
+        img_transforms=Resize(input_size, preserve_aspect_ratio=True),
+    )
+
+    assert len(ds) == 4  # Actual set has 7581382 train and 1337891 test samples
+    image, target = ds[0]
+    assert repr(ds) == f"MJSynth(train={True})"
+    assert isinstance(image, torch.Tensor)
+    assert image.shape[-2:] == input_size
+    assert image.dtype == torch.float32
+    assert isinstance(target['labels'], str)
+
+    loader = DataLoader(ds, batch_size=2, collate_fn=ds.collate_fn)
+    images, labels = next(iter(loader))
+    assert isinstance(images, torch.Tensor) and images.shape == (2, 3, *input_size)
+    assert isinstance(labels, list) and all(isinstance(elt['labels'], str) for elt in labels)
```

Comment:
any way to use `_validate_dataset` instead? to keep a consistent testing structure

Reply:
:+1: